src/promptflow-core/promptflow/core/_model_configuration.py

```diff
@@ -28,6 +28,10 @@ def __post_init__(self):
         self._type = ConnectionType.AZURE_OPEN_AI
         if any([self.azure_endpoint, self.api_key, self.api_version]) and self.connection:
             raise InvalidConnectionError("Cannot configure model config and connection at the same time.")
+        if not self.connection and not all([self.azure_endpoint, self.api_key, self.api_version]):
+            raise InvalidConnectionError(
+                "Missing required params for connection, required azure_endpoint, api_version and api_key."
```

Comment:
Do chatgpt have suggestion on the error message?

Reply:
refine the error message by gpt